Added Quanlse (#71)
qosf/awesome-quantum-software: README.md

**Python**
- [C3](https://github.com/q-optimize/c3) - Open-loop, closed-loop and automated Control, Calibration and Characterization of quantum devices.
- [Krotov](https://github.com/qucontrol/krotov) - Python implementation of Krotov's method for quantum optimal control.

> - [Quanlse](https://github.com/baidu/Quanlse) - Quanlse provides efficient and professional quantum control solutions via an open-source SDK strengthened by cloud service, developed by [Baidu Quantum](https://quantum.baidu.com).

Reviewer: ```suggestion
- [Quanlse](https://github.com/baidu/Quanlse) - Quanlse provides quantum control solutions via a cloud SDK, developed by [Baidu Quantum](https://quantum.baidu.com).
```
